scripts/browserstack/waitfor_tunnel.sh

```diff
@@ -12,6 +13,11 @@ while [ ! -f $BROWSER_PROVIDER_READY_FILE ]; do
     echo
     echo "Timed out after 2 minutes waiting for tunnel ready file"
     exit 5
+  elif [ -f $BROWSER_PROVIDER_ERROR_FILE ]; then
```

Comment:
There's a race condition here, since READY_FILE will still be created when ERROR_FILE exists.  Moving this `IF` outside the `while` will ensure we always take care of this case.